generators/app/templates/_steps.js

```diff
@@ -10,16 +10,33 @@
   express = require('express');
   require('../../spec/factories/index');
 
+  function clearRoutes(_this) {
```

Comment:
this is fine but instead of having to pass in `this` you should just declare this function on the `World` prototype so it already has access to it.